backend/package.json

```
@@ -31,6 +31,7 @@
     "connect-mongodb-session": "2.1.0",
     "cors": "2.8.5",
     "csv-parse": "4.8.5",
+    "device": "^0.3.12",

```

lock the versioning (applies to the other lines you added too)

^ means minimum version, we often end up with compilation problems when the libraries bump their major version and change the apis. so we try to lock down the version and manually upgrade when needed.
```suggestion
    "device": "0.3.12",

```